package.json

```diff
@@ -30,26 +30,28 @@
   },
   "homepage": "https://github.com/statelyai/xstate#readme",
   "dependencies": {
+    "@babel/preset-env": "^7.18.10",
```

Comment:
Added this here because `xstate/vue` tests were complaining that it couldn't be located and installing it in that package directly caused a bunch of test failures. Not really sure what that's about and I don't know anything about `vue`, but this works for some reason